plugins/inputs/kube_inventory/node.go

```diff
@@ -49,5 +49,38 @@ func (ki *KubernetesInventory) gatherNode(n corev1.Node, acc telegraf.Accumulato
 		}
 	}
 
+	convertConditionStatusToNumber := func(status corev1.ConditionStatus) int64 {
+		switch status {
+		case corev1.ConditionTrue:
+			return 1
+		case corev1.ConditionFalse:
+			return 0
+		case corev1.ConditionUnknown:
+			return 2
+		default:
+			return 2
```

Comment:
Would it make sense to convert these to boolean instead of in64? I wonder if that would be easier to use than 0, 1, 2